src/boot.php

```diff
@@ -46,28 +47,45 @@ function() {
 	);
 	add_filter( 'plugin_action_links', 'SeagynDavis\TheSocialLinks\Admin\action_links', 10, 2 );
 
-	add_action( 'admin_enqueue_scripts', 'SeagynDavis\TheSocialLinks\enqueue_admin_scripts' );
+	add_action( 'admin_enqueue_scripts', 'SeagynDavis\TheSocialLinks\enqueue_scripts' );
+	add_action( 'admin_enqueue_scripts', 'SeagynDavis\TheSocialLinks\enqueue_styles' );
 }
 
 /**
- * Enqueue scripts needed for the admin page.
+ * Enqueue front end and admin styles.
  */
-function enqueue_admin_scripts() {
-	wp_enqueue_script( 'jquery-ui-sortable', null, [ 'jquery' ], THE_SOCIAL_LINKS_VERSION, true );
-	wp_enqueue_style( 'font-awesome', THE_SOCIAL_LINKS_URL . 'assets/css/fontawesome.min.css', [], THE_SOCIAL_LINKS_VERSION );
-	wp_enqueue_style( 'font-awesome-brands', THE_SOCIAL_LINKS_URL . 'assets/css/brands.min.css', [ 'font-awesome' ], THE_SOCIAL_LINKS_VERSION );
-	wp_enqueue_style( 'font-awesome-solid', THE_SOCIAL_LINKS_URL . 'assets/css/solid.min.css', [ 'font-awesome' ], THE_SOCIAL_LINKS_VERSION );
+function enqueue_styles() {
+	wp_enqueue_style( 'tsl-font-awesome', THE_SOCIAL_LINKS_URL . 'assets/css/fontawesome.min.css', [], THE_SOCIAL_LINKS_VERSION );
```

Comment:
Sorry, I actually missed this PR. My concern here is that this allows for multiple versions of Font Awesome which will be drain on resources for sites.

Reply:
No worries! I think that's definitely a valid concern. What do you think about checking if Font Awesome has already been loaded, and enqueuing it as a fallback? Something like this:

```php
if ( ( wp_style_is( 'font-awesome', 'enqueued' ) || ( wp_style_is( 'font-awesome', 'registered' ) ) {
    return;
}
	
wp_enqueue_style( 'tsl-font-awesome', THE_SOCIAL_LINKS_URL . 'assets/css/fontawesome.min.css', [], THE_SOCIAL_LINKS_VERSION );
```

Reply:
@seagyn I pulled v2.0.5 earlier and everything is working fine—please disregard my suggestion above!